projects/angular-grid-layout/src/lib/utils/grid.utils.ts

```diff
@@ -21,12 +21,12 @@ export function ktdGridCompact(layout: KtdGridLayout, compactType: KtdGridCompac
         .map(item => ({ id: item.id, x: item.x, y: item.y, w: item.w, h: item.h, minW: item.minW, minH: item.minH, maxW: item.maxW, maxH: item.maxH }));
 }
 
-function screenXPosToGridValue(screenXPos: number, cols: number, width: number): number {
-    return Math.round((screenXPos * cols) / width);
+function gridXPosToGridValue(screenXPos: number, cols: number, width: number, gap: number): number {
+    return Math.round(((screenXPos + gap) * cols) / width);
 }
 
-function screenYPosToGridValue(screenYPos: number, rowHeight: number, height: number): number {
-    return Math.round(screenYPos / rowHeight);
+function gridYPosToGridValue(screenYPos: number, rowHeight: number, height: number, gap: number): number {
+    return Math.round((screenYPos + gap) / rowHeight + gap);
```

Comment:
I may miss something about this solution, but I think it does not fix the issue when the gap is really big, waht about something really simple like:
`return Math.round(screenYPos / (rowHeight + gap));`

It is not 100% accurate, but would work for now. I am thinking in completely change this function, so for now this would be enough.

Reply:
I don't know why I had that screenYPos + gap know that I am relooking it as it definitely does not work. I will take that + gap out as you suggested.

Reply:
Ope actually I think I missed some parenthesis as I now see why that was added...
`return Math.round((screenYPos + gap) / (rowHeight + gap))` 

This allows for the bottom of the placeholder to show when the gap is big relative to the rowHeight (say 100 gap to 50 rowHeight). Without this, resizing downwards forces the user to scroll past the desired row they want to resize into for the placeholder to scale correctly. Its not a huge issue, just visually seemed a bit off, but I can go with your solution if you prefer it.